.babelrc.js

```diff
@@ -6,10 +6,8 @@ if (process.env.NODE_ENV !== `test`) {
   ignore.push(`**/__tests__`)
 }
 
-const presetAbsPath = require(`path`).join(__dirname, '.babel-preset.js')
-
 module.exports = {
   sourceMaps: true,
-  presets: [presetAbsPath],
+  presets: ["gatsby"],
```

Comment:
This is more of a personal preference, but can we always refer to the _whole_ path? I've always liked being more explicit!

e.g. `presets: ['babel-preset-gatsby']`